content/projects/kotlin/liner-layout-using-the-layout-editor/_index.md

```diff
@@ -0,0 +1,495 @@
+---
+title:  Linear layout using the Layout Editor
+ready: True
+prerequisites:
+  hard: ["topics/kotlin/grid-layout", "topics/kotlin/basic-control-flow", "topics/kotlin/basic-syntax-types"]
```

Comment:
topics/kotlin/grid-layout does not exist here so the content link does not work but it may be in another PR?

Reply:
Yes it is in the andriod conternt one